bookish/Services/list_services.py

```diff
@@ -0,0 +1,47 @@
+from bookish.models import db
+from bookish.models.users import Users
+from bookish.models.books import Books
+
+
+def list_all_users():

```

Comment:
`list` tipically means `print`, but this isn't suggestive for what this is doing. 

Hint: think of what type of method you use this function for

Reply:
Reminder